### DISCUSSION TITLE
In Debug Commands

### DISCUSSION THREAD

**Discussion by JustaSissy** [zippity21/Stardew_Valley_Regression_Mod#125] [Q&A]
Hi I'm new to stardew valley modding and I saw that there are commands under "in debug" that affect certain stats however I'm not sure how to execute them. How do I enter the debug mode or what do I do to do them? Every time I press "alt + f1" for example it just makes me wet

**Comment by GardensStarlit** [comment]
If you are able to see the bowel and bladder meters as well as food/water, and press f3 to open a menu with underwear types, then you're in debug mode. 

Debug mode can be toggled on and off with F9 — and then you should be able to use the alt and shift-alt commands as expected.
